src/sedpack/io/flatbuffer/iterate.py

```diff
@@ -82,18 +82,16 @@ def _iterate_content(self, content: bytes) -> Iterable[ExampleT]:
                     attribute=attribute,
                 )
 
-                # Copy otherwise the arrays are immutable and keep the whole
-                # file content from being garbage collected.
-                np_array = np.copy(np_array)
```

Comment:
Copy issues should be fine since Rust is allocating the memory and
passing ownership to Python.